corehq/apps/enterprise/enterprise.py

```diff
@@ -685,3 +690,30 @@ def rows_for_domain(self, domain_obj):
         if domain_obj.two_factor_auth:
             return []
         return [(domain_obj.name,)]
+
+
+class EnterpriseDataForwardingReport(EnterpriseReport):
+    title = gettext_lazy('Data Forwarding')
+    total_description = gettext_lazy('# of Data Forwarders')
```

Comment:
Since https://github.com/dimagi/commcare-hq/pull/35622 is merged, do you want to remove the `total_description` of this tile? I'm not sure if this statistics is obvious enough when the title is "Data Forwarding"

Reply:
Addressed in e2e0524e80710dfe91164ba877dfde8ab75513c2

Reply:
You convinced me the other way :-) Reverting this change, as its probably intuitive to the user that a "Data Forwarding Report" is going to show the number of data forwarders vs the number of forwarded records.